app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<h1 class="text-center">jobs dashboard</h1>
+
+<div class="row">
+  <div class="col-md-4 dashboard">
+    <div class="category">
+      <%= link_to "jobs by location", "/jobs?sort=location", class: "job-name" %>
+    </div>
+  </div>
+  <div class="col-md-4 dashboard">
+    <div class="category">
+      <%= link_to "jobs by interest level", "/jobs?sort=interest", class: "job-name" %>
+    </div>
+  </div>
+  <div class="col-md-4 dashboard">
+    <div class="category">
+      <%= link_to "jobs in Denver", "/jobs?location=Denver", class: "job-name" %>
+    </div>
+  </div>
+</div>
+
+<h2 class="text-center">top companies by level of interest</h2>
+
+<ul>
+  <% @payload[:top_companies].each do |key, value| %>
```

Comment:
i'd choose better names for `key` and `value` - what do these variable represent?